galaxy/wrapper/test-data/profiler_result2.tabular

```diff
@@ -0,0 +1,3 @@
+bin labels		-0.0Kb	0.0Kb
```

Comment:
It turns out that the `--outFileNameData` option is returning the data for the profile on the top, while `--outFileNameMatrix` is returning the data for the heatmap on the bottom.